client/app/docs/module.js

```diff
@@ -34,6 +34,31 @@
                 scope.closeModal = function() {
                     scope.isModalOpen = false;
                 };
+
+                //Select boxes
+                scope.opts = ['Serbia', 'Czech Republic', 'Germany', 'Australia'];
+
+                //Typeahead
+                scope.taTerms = ['Serbia', 'Czech Republic', 'Germany', 'Australia', 'Canada', 'Russia', 'Italy', 'Egypt', 'China'];
```

Comment:
I don't know what is the other option? It's just an example how to use autocomplete element...